guides/2022-04-05-conduit-kafka-connect-connector.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Using Kafka Connect Connectors with Conduit"
+slug: using-kafka-connect-connectors-with-conduit
+author: '@anaptfox'
+author_url: https://twitter.com/anaptfox
+author_image_url: https://pbs.twimg.com/profile_images/1004707739405205504/BpvzOr9X_400x400.jpg
+tag: ['File Connector']
+description: Learn how to build a Conduit Algolia Connector
+keywords: ['data', 'connector', 'algolia', 'conduit']
```

Comment:
There appear to have sneaked in from another page. We can probably replace file and algolia with Kafka Connect.